.travis.yml

```diff
@@ -83,7 +83,6 @@ matrix:
             - python3-dev
             - tcl
             - libsvm-dev
-            - libpoppler-cpp-dev
```

Comment:
Presumably this change is accidental?

If it's actually wanted (because poppler isn't used in this case or something like that) then it doesn't seem to belong here and should really be submitted as a separate patch.

Reply:
It was a mistake. It should not be there.